srcpkgs/python-language-server/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'python-language-server'
+pkgname=python-language-server
+version=0.31.4
+revision=1
+build_style=python-module
+pycompile_module="pyls"
+short_desc="Python 2 implementation of the Language Server Protocol"
+maintainer="k4leg <d0xi@inbox.ru>"
+license="MIT"
+homepage="https://github.com/palantir/python-language-server"
+distfiles="${homepage}/archive/${version}.tar.gz"
+checksum=3555ebe496858f896eb32c713ce244d5897da2be52a9c17d199ec4026371ad43
+
+if [ "$CROSS_BUILD" ]; then
```

Comment:
This seems wrong

Reply:
What exactly? Everything seems to be in order.

Reply:
Even if this was correct, you could just make it hostmakedepends always (when not cross compiling, hostmakedepends and makedepends is the same). But as it seems to me, stuff like setuptools should always be hostmakedepends, while "target" dependencies should be makedepends.

Reply:
Yes, thank you!